README.md

```diff
@@ -94,24 +122,24 @@ struct YourApp: App {
 }
 ```
 
-You also need to disable swzzling by setting `configuration.isTrackScreenViewEvents = false`, see the next configuration steps.
+You also need to disable swzzling by setting `configuration.withTrackScreenViewEvents(false)`, see the next configuration steps.
```

Comment:
method swizzling is a feature in Swift that allows us to replace the implementation of os methods like `viewDidAppear` so that we can automatically track screen view events.

I will change this to make it easier to understand.